snmp/datadog_checks/snmp/data/default_profiles/netgear-readynas.yaml

```diff
@@ -0,0 +1,126 @@
+extends:
+  - _generic-ucd.yaml
+  - netgear-generic.yaml
+sysobjectid:
+  - 1.3.6.1.4.1.4526.100.12.*
+  - 1.3.6.1.4.1.4526.100.15.*
+  - 1.3.6.1.4.1.4526.100.16.*
+  - 1.3.6.1.4.1.4526.22.*
+metrics:
+  - MIB: READYNASOS-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.4526.22.1.0
+      name: netgear.nasMgrSoftwareVersion
+      tag: netgear_nas_mgr_software_version
+  - MIB: READYNASOS-MIB
+    table:
+      OID: 1.3.6.1.4.1.4526.22.3
+      name: diskTable
+    symbols:
+      - OID: 1.3.6.1.4.1.4526.22.3.1.6
+        name: netgear.ataError
+      - OID: 1.3.6.1.4.1.4526.22.3.1.7
+        name: netgear.diskCapacity
+      - OID: 1.3.6.1.4.1.4526.22.3.1.10
+        name: netgear.diskTemperature
+    metric_tags:
+      - tag: netgear_disk_id
+        column:
+          OID: 1.3.6.1.4.1.4526.22.3.1.2
+          name: netgear.diskID
+      - tag: netgear_disk_slot_name
+        column:
+          OID: 1.3.6.1.4.1.4526.22.3.1.3
+          name: netgear.diskSlotName
+      - tag: netgear_disk_serial
+        column:
+          OID: 1.3.6.1.4.1.4526.22.3.1.4
+          name: netgear.diskSerial
+      - tag: netgear_disk_model
+        column:
+          OID: 1.3.6.1.4.1.4526.22.3.1.5
+          name: netgear.diskModel
+      - tag: netgear_disk_interface
+        column:
+          OID: 1.3.6.1.4.1.4526.22.3.1.8
+          name: netgear.diskInterface
+      - column:
+          OID: 1.3.6.1.4.1.4526.22.3.1.9
+          name: netgear.diskState
+        tag: netgear_disk_state
+        mapping:
+          1: online
+          0: offline
+  - MIB: READYNASOS-MIB
+    table:
+      OID: 1.3.6.1.4.1.4526.22.4
+      name: fanTable
+    symbols:
+      - OID: 1.3.6.1.4.1.4526.22.4.1.2
+        name: netgear.fanRPM
+      - OID: 1.3.6.1.4.1.4526.22.4.1.3
+        name: netgear.fanStatus
+    metric_tags:
+      - tag: netgear_fan_type
+        column:
+          OID: 1.3.6.1.4.1.4526.22.4.1.4
+          name: netgear.fanType
+  - MIB: READYNASOS-MIB
+    table:
+      OID: 1.3.6.1.4.1.4526.22.5
+      name: temperatureTable
+    symbols:
+      - OID: 1.3.6.1.4.1.4526.22.5.1.2
+        name: netgear.temperatureValue
+      - OID: 1.3.6.1.4.1.4526.22.5.1.4
+        name: netgear.temperatureMin
+      - OID: 1.3.6.1.4.1.4526.22.5.1.5
+        name: netgear.temperatureMax
+    metric_tags:
+      - tag: netgear_temperature_type
+        column:
+          OID: 1.3.6.1.4.1.4526.22.5.1.3
+          name: netgear.temperatureType
+  - MIB: READYNASOS-MIB
+    table:
+      OID: 1.3.6.1.4.1.4526.22.7
+      name: volumeTable
+    symbols:
+      - OID: 1.3.6.1.4.1.4526.22.7.1.3
+        name: netgear.volumeRAIDLevel
```

Comment:
similarily to `fanStatus`, this also is of type string, you can move it to the metric_tags instead